addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -245,8 +245,12 @@ const PopupWidget = publicWidget.Widget.extend({
     /**
      * @private
      */
-    _onHashChange() {
-        this._showPopupOnClick();
+    _onHashChange(ev) {
+        // Keep the new hash from the event to avoid conflict with the eCommerce
+        // hash attributes managing.
+        // TODO : it should not have been a hash at all for ecommerce, but a
+        // query string parameter
+        this._showPopupOnClick(new URL(ev.newURL).hash);
```

Comment:
This will crash if someone is calling this method without passing an `ev` parameter, we have to protect that case as we are in stable.